lambdas/process-candidate-identity/src/main/java/uk/gov/di/ipv/core/processcandidateidentity/ProcessCandidateIdentityHandler.java

```
@@ -311,6 +312,7 @@ private CoiCheckType getCoiCheckType(
         return CoiCheckType.STANDARD;
     }
 
+    @SuppressWarnings("java:S3776") // Cognitive Complexity of methods should not be too high
```

I'm not keen on suppressing these errors. If we think it's too strict we should consider upping the limit, otherwise we should look at making this method easier to follow.